src/lib/containers/entity/metadata/EntityModal.tsx

```diff
@@ -35,21 +36,26 @@ export const EntityModal: React.FC<EntityModalProps> = ({
   showTabs = true,
 }: EntityModalProps) => {
   const [currentTab, setCurrentTab] = useState<EntityModalTabs>(initialTab)
-  const { data: entityBundle, isLoading: isLoadingBundle } = useGetEntityBundle(
-    entityId,
-  )
+  const [isInEditMode, setIsInEditMode] = useState(false)
+  const { data: entityBundle } = useGetEntityBundle(entityId)
 
   return (
     <Modal
       className="bootstrap-4-backport EntityMetadata"
+      backdrop={'static'}
+      size={isInEditMode ? 'lg' : undefined}
       show={show}
       animation={false}
       onHide={onClose}
     >
       <Modal.Header closeButton>
-        <Modal.Title>
-          {isLoadingBundle ? <SynapseSpinner /> : entityBundle?.entity?.name}
-        </Modal.Title>
+        {entityBundle ? (
+          <Modal.Title>{entityBundle.entity!.name}</Modal.Title>
+        ) : (
+          <Skeleton>
+            <Modal.Title>Placeholder</Modal.Title>
```

Comment:
From the title, it looks like this text is not actually rendered (but only used for the string length).  Is that right?

Reply:
Exactly!

Reply:
Ideally I wouldn't want to see placeholder dummy text all over the project (if we adopt this UI, which I think we should since it's better than a spinner!).